autoload/lsp/lspserver.vim

```diff
@@ -347,6 +347,14 @@ def ServerInitReply(lspserver: dict<any>, initResult: dict<any>): void
   var caps: dict<any> = initResult.capabilities
   lspserver.caps = caps
 
+  for [key, val] in initResult->items()
+    if key == 'capabilities'
+      continue
+    endif
+
+    lspserver.caps[$'~additionalInitResult_{key}'] = val
```

Comment:
This will add a bit of extra information to ":LspShowServerCapabilities":

Notice the last two lines

```
'clangd' Language Server Capabilities
=====================================
astProvider: true
callHierarchyProvider: true
codeActionProvider: {'codeActionKinds': ['quickfix', 'refactor', 'info']}
compilationDatabase: {'automaticReload': true}
completionProvider: {'allCommitCharacters': [' ', '	', '(', ')', '[', ']', '{', '}', '<', '>', ':', ';', ',', '+', '-', '/', '*', '%', '^', '&', '#', '?', '.', '=', '"', '''', '|'], 'resolveProvider': false, 'triggerCharacters': ['.', '<', '>', ':', '"', '/', '*']}
declarationProvider: true
definitionProvider: true
documentFormattingProvider: true
documentHighlightProvider: true
documentLinkProvider: {'resolveProvider': false}
documentOnTypeFormattingProvider: {'moreTriggerCharacter': [], 'firstTriggerCharacter': ' '}
documentRangeFormattingProvider: true
documentSymbolProvider: true
executeCommandProvider: {'commands': ['clangd.applyFix', 'clangd.applyTweak']}
hoverProvider: true
implementationProvider: true
memoryUsageProvider: true
referencesProvider: true
renameProvider: true
selectionRangeProvider: true
semanticTokensProvider: {'full': {'delta': true}, 'legend': {'tokenTypes': ['variable', 'variable', 'parameter', 'function', 'method', 'function', 'property', 'variable', 'class', 'interface', 'enum', 'enumMember', 'type', 'type', 'unknown', 'namespace', 'typeParameter', 'concept', 'type', 'macro', 'comment'], 'tokenModifiers': ['declaration', 'deprecated', 'deduced', 'readonly', 'static', 'abstract', 'virtual', 'dependentName', 'defaultLibrary', 'usedAsMutableReference', 'functionScope', 'classScope', 'fileScope', 'globalScope']}, 'range': false}
signatureHelpProvider: {'triggerCharacters': ['(', ',']}
textDocumentSync: {'save': true, 'change': 2, 'openClose': true}
typeHierarchyProvider: true
workspaceSymbolProvider: true
~additionalInitResult_offsetEncoding: 'utf-32'
~additionalInitResult_serverInfo: {'version': 'Apple clangd version 14.0.0 (clang-1400.0.29.202) mac+xpc x86_64-apple-darwin22.3.0; target=arm64-apple-darwin22.3.0', 'name': 'clangd'}
```